index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <nav>
+        <a href="#"><h1>My Logo</h1></a>
+        <a href="#">
+            <img src="images/Union.png" alt="Union Image">
+        </a>
+    </nav>
```

Comment:
- [ ] Good job on creating the Navbar 👍 however according to the requirements The container of this section should be a header tag `<header>`

Reply:
Done.